ssds/storage.py

```diff
@@ -71,6 +72,10 @@ def _do_oneshot_copy():
             self._async_set.put(self._finalize_copy, src_blob, dst_blob, tags)
 
     def _download(self, src_blob: AnyBlob, dst_blob: LocalBlob):
+        dirname = os.path.dirname(dst_blob.url)
```

Comment:
Should there be a sanity check here for `s3://` type cloud prefixes?

For example:

```
>>> print('ok' if os.path.dirname('s3://home/quokka/j') else 'not ok')
ok
```

Reply:
Currently blobs are instantiated from `blob_for_url`, which prevents this from happening.

Possibly there could be sanity checking in `ssds.blobstore.local` to prevent Python API users from stepping on their own toes.